classes/Tools.php

```diff
@@ -4415,11 +4415,35 @@ public static function isFileFresh(string $file, int $timeout = self::CACHE_LIFE
 
         return false;
     }
+
+    /**
+     * @return bool
+     */
+    public static function isDetectCountryFromBrowserLanguageAvailable(): bool
+    {
+        $languages = null;
+
+        return Configuration::get('PS_DETECT_COUNTRY') && isset($_SERVER['HTTP_ACCEPT_LANGUAGE'])
+            && self::isValidAcceptLanguageAvailable($languages)
+            && Validate::isLanguageIsoCode($languages[0]);
```

Comment:
Maybe we should verify `$languages` contain a offset 0?

Reply:
I ve change the cinematic to avoid use of by reference parameters